IntelFsp2WrapperPkg/IntelFsp2WrapperPkg.dec

```diff
@@ -137,3 +138,7 @@
     #1: measure FSP UPD region in one record (PCR1), the FSP code without UPD in another record (PCR0).
   #
   gIntelFsp2WrapperTokenSpaceGuid.PcdFspMeasurementConfig|0x00000000|UINT32|0x50000004
+  #
+  # Non-0 means PcdFspiUpdDataAddress will be ignored, otherwise PcdFspiUpdDataAddress will be used.

```

Comment:
May I know why the PCD will be ignored when it's non-0?

Reply:
no need this description, because fspi just have PcdFspiUpdDataAddress, not have PcdFspiUpdDataAddress64, so it should not like PcdFspsUpdDataAddress64